### PR TITLE
Updated Intro_Sorting.mdx with python soln

### PR DESCRIPTION
**Pull request by 19WH1A0524**
Updated Intro_Sorting.mdx with python solution of hacker rank

_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

**Comment by SansPapyrus683** [comment]
uh could you make the code kinda like the c++/java versions?

**Comment by 19WH1A0524** [comment]
Its actually the solution for the problem mentioned in the page

File: content/2_Bronze/Intro_Sorting.mdx [hunk 1]
```diff
@@ -173,6 +173,38 @@ Two ways to avoid this:
...
+    for j in range(0, n - 1):
+        if a[j] > a[j + 1]:
+            a[j], a[j + 1] = a[j + 1], a[j]
+            currentSwaps += 1
+            
+    numSwaps += currentSwaps
+            
+    if currentSwaps == 0:
+        break
+
```
Comment (SansPapyrus683) [inline content/2_Bronze/Intro_Sorting.mdx:199]: please make consistent with c++ and java sols

File: content/2_Bronze/Intro_Sorting.mdx [hunk 2]
```diff
@@ -173,6 +173,38 @@ Two ways to avoid this:
...
 ### Dynamic Arrays
+<PySection>
+
+
+
+```py
+import sys
+
+n = int(input())
+a = [int(i) for i in input().strip().split(' ')]
```
Comment (chezbgone) [inline content/2_Bronze/Intro_Sorting.mdx:184]: The `str.split()` method has a default argument of `' '` already, so you don't need it here.

File: content/2_Bronze/Intro_Sorting.mdx [hunk 3]
```diff
@@ -173,6 +173,38 @@ Two ways to avoid this:
 </LanguageSection>
 
 ### Dynamic Arrays
+<PySection>
+
+
+
+```py
+import sys
```
Comment (chezbgone) [inline content/2_Bronze/Intro_Sorting.mdx:181]: You don't seem to use this import anywhere?

File: content/2_Bronze/Intro_Sorting.mdx (hunk 1 above)
Comment (chezbgone) [reply content/2_Bronze/Intro_Sorting.mdx:199]: In Python, we usually just use the `list.sort` method or the `sorted` function, which is both more readable and more efficient than implementing it manually.

**Comment by chezbgone** [comment]
> Its actually the solution for the problem mentioned in the page

In this case, you want to put the solution right after the problem, inside of a spoiler tag. There is an example of this on line 35 of https://raw.githubusercontent.com/cpinitiative/usaco-guide/master/content/2_Bronze/Simulation.mdx using `<Spoiler title="Solution">`

**Pull request closed without merging**